sdk/pinocchio/src/sysvars/fees.rs

```diff
@@ -74,7 +74,7 @@ impl FeeRateGovernor {
 }
 
 /// Fees sysvar
-#[derive(Debug)]
+#[derive(Debug, Default, Copy, Clone)]
```

Comment:
Is there a motivation for adding `Default` here?
```suggestion
#[derive(Clone, Copy, Debug, Default)]
```